docs/rest.md

```diff
@@ -138,8 +142,12 @@ Reserved for human-readable content describing the service.
 
 ```javascript
 {
- "flavor": "core",
  "series": "16",
+ "version": "2.0.17",
+ "os-release": {
+   "id": "ubuntu",
+   "version-id": "17.04",
+ },
  "store": "store-id"          // only if not default
```

Comment:
I couldn't see "store" being set anywhere in the snapd code (could be wrong here).
This is missing "on-classic".